java/src/jmri/jmrix/bachrus/kpfserialdriver/README

```diff
@@ -0,0 +1,6 @@
+This is the README file for the jmrix.bachrus.serialdriver package.
+The jmrix Java package defines implementations for the JMRI interfaces.
+jmrix.bachrus.serialdriver defines classes for interfacing to a Bachrus
```

Comment:
Shouldn't this be `jmri.jmrix.bachrus.serialdriver`? It seems `jmri.` is missing at the beginning.

Reply:
I generally only write out enough to make sure that the name is unambiguous, as our package names get a little long.  Here. I've moved the README (which is a deprecated approach ) to a package info file to fix it.